src/components/button/AposButton.stories.js

```diff
@@ -40,21 +42,11 @@ storiesOf('Button', module)
     `,
     ...mixin
   }))
-  .add('Default, Busy', () => ({
```

Comment:
Deprecated. The button's business is now its own business (English is fun).

Reply:
Disabled can be controlled by the context, though. As it is in the previous one.